content/blog/zenbleed-patch-call-for-testing.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
+type: blog
+author: 
+ name: "Andrew Lukoshko"
+ bio: "Release Engineering Lead"
+ image: /users/alukoshko.jpg
+date: '2023-07-24'
+post:
+    title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
+    image: /blog-images/23.07.24.zenbleed.png
```

Comment:
```suggestion
  title: "Testers needed: Zenbleed patch for AlmaLinux 8 and 9"
  image: /blog-images/23.07.24.zenbleed.png
```

Reply:
I'm hesitant to remove those spaces, as they've been in all of our other blog posts, and I know md can get picky about spaces.

Reply:
It shouldn't matter either way I guess, I just like consistency :)

Reply:
Yeah, it's consistently got spaces, so we'll keep it for now. Those are meant to be the opengraph details that get shared... when we get that fixed. :D